backend/core/src/listings/listings.controller.ts

```diff
@@ -100,12 +113,25 @@ export class ListingsController {
     @Param("listingId") listingId: string,
     @Body() listingUpdateDto: ListingUpdateDto
   ): Promise<ListingDto> {
-    return mapTo(ListingDto, await this.listingsService.update(listingUpdateDto))
+    const listing = await this.listingsService.update(listingUpdateDto)
+    /**
+     * clear list caches
+     * As we get more listings we'll want to update this to be more selective in clearing entries
+     */
+    for (const key of this.cacheKeys) {
+      await this.cacheManager.del(key)
+    }
+    await this.cacheManager.del(`/listings/${listingId}`)
+    return mapTo(ListingDto, listing)
   }
 
   @Delete(`:listingId`)
   @ApiOperation({ summary: "Delete listing by id", operationId: "delete" })
   async delete(@Param("listingId") listingId: string) {
     await this.listingsService.delete(listingId)
+    for (const key of this.cacheKeys) {
```

Comment:
This deletion of cacheKeys + listingId related key could be a private method because it's exactly the same code in every handler using it.

Reply:
I agree. Before I create any methods though, I want us to come to an agreement on how we should handle posts/puts for the list route caches. Since we have such few listings I think it's fine to clear those routes from the cache, but it's easy enough now to add in handling it by updating the entries. In the case of `post`, this is relatively easy, we get the current entry, add the new listing and set it. In `put` there's an extra step of making sure we filter out the old listing from the cached route before adding it. This all becomes more involved when paging and filtering get added, where we'd want to only clear/update the routes that the listing is in, which we could also accommodate now, or just iterate when we need to.

Reply:
> Since we have such few listings I think it's fine to clear those routes from the cache 
I'm OK with that too.